demos/accordion/index.php

```diff
@@ -0,0 +1,208 @@
+
+	<div role="main" class="ui-content jqm-content">
+
+		<h1>Accordion <a href="http://api.jquerymobile.com/accordion/" class="jqm-api-docs-link ui-button ui-nodisc-icon ui-alt-icon ui-button-inline ui-corner-all ui-mini">API <span class="ui-icon ui-icon-caret-r"></span></a></h1>
+
+
+		<p>Displays collapsible content panels for presenting information in a limited amount of space.</p>
+
+		<h2>Default Accordion</h2>
+
+		<p>Click headers to expand/collapse content that is broken into logical sections, much like tabs. Optionally, toggle sections open/closed on mouseover.</p>
+
+		<p>The underlying HTML markup is a series of headers (H3 tags) and content divs so the content is usable without JavaScript.</p>
+
+		<div data-demo-html="true">
+				<div data-role="accordion">
+					<h3>Section 1</h3>
+					<div>
+						<p>
+							Mauris mauris ante, blandit et, ultrices a, suscipit eget, quam. Integer ut neque. Vivamus nisi metus, molestie vel, gravida in, condimentum sit amet, nunc. Nam a nibh. Donec suscipit eros. Nam mi. Proin viverra leo ut odio. Curabitur malesuada. Vestibulum a velit eu ante scelerisque vulputate.
+						</p>
+					</div>
+					<h3>Section 2</h3>
+					<div>
+						<p>
+							Sed non urna. Donec et ante. Phasellus eu ligula. Vestibulum sit amet purus. Vivamus hendrerit, dolor at aliquet laoreet, mauris turpis porttitor velit, faucibus interdum tellus libero ac justo. Vivamus non quam. In suscipit faucibus urna.
+						</p>
+					</div>
+					<h3>Section 3</h3>
+					<div>
+						<p>
+							Nam enim risus, molestie et, porta ac, aliquam ac, risus. Quisque lobortis. Phasellus pellentesque purus in massa. Aenean in pede. Phasellus ac libero ac tellus pellentesque semper. Sed ac felis. Sed commodo, magna quis lacinia ornare, quam ante aliquam nisi, eu iaculis leo purus venenatis dui.
+						</p>
+						<ul>
+							<li>List item one</li>
+							<li>List item two</li>
+							<li>List item three</li>
+						</ul>
+					</div>
+					<h3>Section 4</h3>
+					<div>
+						<p>
+							Cras dictum. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Aenean lacinia mauris vel est.
+						</p>
+						<p>
+							Suspendisse eu nisl. Nullam ut libero. Integer dignissim consequat lectus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+						</p>
+					</div>
+				</div>
+		</div><!--/demo-html -->
+
+		<h2>Fill Space</h2>
+		<p>Because the accordion is comprised of block-level elements, by default its width fills the available horizontal space. To fill the vertical space allocated by its container, set the <code>data-height-style</code> option to <code>"fill"</code>, and the script will automatically set the dimensions of the accordion to the height of its parent container.</p>
+		<div data-demo-html="true">
+				<div data-role="accordion" data-height-style="fill">
```

Comment:
Too much indent though you can use it because you need to add a parent container with a height set for this option to work properly.